webui/react/src/services/utils.ts

```diff
@@ -72,7 +76,15 @@ export const processApiError = (name: string, e: unknown): void => {
     options.publicSubject = `Failed in decoding ${name} API response.`;
   }
 
-  handleError(e, options);
+  if (isApiResponse(e)) {
+    try {
+      const response = await e.json();
+      options.publicMessage = response.message || response.error?.error;
+    } catch (err) {
+      // Ignore the error.
```

Comment:
**question:** does the comment make eslint treat it as non-empty?

Reply:
I should probably make the comment more useful though. why we ignore it

Reply:
maybe we shouldn't ignore and somehow encode it in the returned DetError